package/template-cartesian-graph/package

```diff
@@ -16,9 +16,9 @@ source=("https://github.com/PeterGrace/template-cartesian-graph/archive/v${pkgve
 sha256sums=(d3215546c6fd709bb5988b7042ce5fb990d9a030d783ed7505fcc2dbedbc0e0f)
 
 package() {
-    install -D -m 755 -t "$pkgdir"/opt/share/remarkable/templates \
```

Comment:
templatectl handles copying these files into the correct place. This will need to be reverted as we don't want this package to install to the root partition and fill it up.

Reply:
Should be fixed.